packages/manager/apps/web-domains/src/alldoms/components/DatagridColumns/DatagridColumnServiceName.tsx

```diff
@@ -0,0 +1,27 @@
+import { useNavigationGetUrl } from '@ovh-ux/manager-react-shell-client';
+import { DataGridTextCell } from '@ovh-ux/manager-react-components';
+import React from 'react';
+import { OdsLink } from '@ovhcloud/ods-components/react';
+import { TServiceDetail } from '@/alldoms/types';
+import config from '@/web-domains.config';
+
+interface DatagridColumnServiceNameProps {
+  readonly props: TServiceDetail;
+}
+
+export default function DatagridColumnServiceName({
+  props,
+}: DatagridColumnServiceNameProps) {
+  const { domain } = props.serviceInfo;
+  const { data: url } = useNavigationGetUrl([
+    config.rootLabel,
+    `/alldoms/${domain}`,
+    {},
+  ]);
+
+  return (
+    <DataGridTextCell>
+      <OdsLink href={`${url}`} label={domain} data-testid={domain}></OdsLink>
```

Comment:
```suggestion
      <OdsLink href={url} label={domain} data-testid={domain}></OdsLink>
```
isn't url already a string ?

Reply:
url is not typed and href wait a string
but href={url as string} sould be work

Reply:
I think it's better to type url if it isn't typed